src/main/java/seedu/address/model/time/TimeMask.java

```diff
@@ -101,6 +110,21 @@ private void occupySlots(int dayIndex, int startHourIndex, int endHourIndex) {
         weeklyOccupancy[dayIndex] = weeklyOccupancy[dayIndex] | mask;
     }
 
+    private void occupyMultipleDay(int startDay, int endDay, int starHour) {
+        int curr = startDay;
+        int startBits = Integer.parseInt("1".repeat(23 - starHour + 1), 2);
+        int mask = startBits << starHour;
+        weeklyOccupancy[curr] = weeklyOccupancy[curr] | mask;
+        curr = curr == 6 ? 0 : curr + 1;
+
+        while (curr < endDay) {
```

Comment:
What if we start on Friday (index 5) and end on Tuesday (index 2)? Would that cause bugs? Just checking